internal/iam/query.go

```diff
@@ -266,5 +266,120 @@ const (
 	from iam_role_project
 	where public_id = any(select role_id from combined_role_types)
     order by update_time desc, public_id desc
-	`
+`
+	roleGrantsScopeQuery = `
+	with 
+	global_roles (role_id) as (
+		select
+			public_id as role_id
+		from iam_role_global
+		where public_id = any($1)
```

Comment:
The formatting throughout this entire query is a bit off in places. For example this CTE should be:

```suggestion
        select public_id as role_id
          from iam_role_global
         where public_id = any($1)
```

Can you update it to follow the [sql style guide](https://github.com/hashicorp/boundary/blob/main/sql-style-guide.md#data-manipulation-statements)?

Reply:
done 👍. I also went removed the leading "tab"